### PR TITLE
feat: added quiz question component

### PR DESCRIPTION
**Pull request by eakansh125**

### CODE REVIEW

**Review by Sahil25061999 — APPROVED** [review]
Mostly looks good to me with some minor issues ( px, rem ), a suggestion for UI use secondary button styling for previous button.

**Comment by eakansh125** [comment]
> Mostly looks good to me with some minor issues ( px, rem ), a suggestion for UI use secondary button styling for previous button.

That's a really good suggestion. Will keep that in mind!!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                <a class="text-dec-none font-bold margin-lr font-primary-color" href="#">Home</a>
+                <a class="text-dec-none font-bold margin-lr font-primary-color" href="screens/rules.html">Rules</a>
+            </div>
+            <div class="nav-header-div-mid">
+                <input class="input-outlined" type="text" placeholder="Search" />
+            </div>
+            <div class="nav-header-div-rt">
+
+                <a href="#" class="btn-small margin-lr">Login</a>
+                <img class="pointer" src="https://img.icons8.com/material/36/000000/user-male-circle--v1.png" />
```
Comment (pradipgharge) [inline index.html]: alt is missing for the icon

File: index.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            <div class="nav-header-div-lt">
+                <a class="text-dec-none font-bold margin-lr font-primary-color" href="#">Home</a>
+                <a class="text-dec-none font-bold margin-lr font-primary-color" href="screens/rules.html">Rules</a>
+            </div>
+            <div class="nav-header-div-mid">
+                <input class="input-outlined" type="text" placeholder="Search" />
+            </div>
+            <div class="nav-header-div-rt">
+
+                <a href="#" class="btn-small margin-lr">Login</a>
```
Comment (pradipgharge) [inline index.html:23]: link missing here in a tag

File: screens/category.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    border-radius: 10px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    flex-wrap: wrap;
+}
+
+.category-card{
+    width: 250px;
+    height: 420px;
```
Comment (pradipgharge) [inline screens/category.css]: use rem for length units

File: screens/quiz.css [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    width: 80%;
+    display: flex;
+    justify-content: space-between;
+}
+
+.quiz-body{
+    width: 80%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
```
Comment (pradipgharge) [inline screens/quiz.css:24]: You can add utility class for this

File: style.css [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+    width: 100%;
+}
+
+/* Nav */
+
+.nav-header{
+    position: sticky;
+    top: 0;
+    height: auto;
+    /* box-shadow: inset 0px -1px 1px #e7ebf0; */
```
Comment (pradipgharge) [inline style.css]: Remove commented code

File: style.css [hunk 6]
```diff
@@ -0,0 +1,264 @@
...
+
+.btn-small:hover{
+    background-color: var(--dark-maroon);
+    border: 1px solid var(--dark-maroon);
+    font-weight: bold;
+}
+
+/* Main */
+
+
```
Comment (pradipgharge) [inline style.css]: remove extra spacing here

**Review by pradipgharge — APPROVED** [review]
Suggested few changes, good to merge after correcting them

File: index.html (hunk 1 above)
Comment (eakansh125) [reply index.html:25]: > Suggested few changes, good to merge after correcting them

Will do all the requested changes at once! Thanks for the review!!

**Comment by eakansh125** [comment]
> Suggested few changes, good to merge after correcting them

Will do all the requested changes at once! Thanks for the review!!

**Pull request merged**